packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -124,17 +155,10 @@ export const tour_styles_dark_mode: Styles = {
     },
 };
 
-export const tour_step_locale: Locale = {
-    back: <SpanButton has_effect text={localize('Repeat tour')} secondary medium />,
-    close: localize('Close'),
-    last: localize('OK'),
-    next: localize('Next'),
-    skip: localize('Skip'),
-};
-
-export const eu_tour_step_locale: Locale = {
-    close: localize('Close'),
-    last: localize('OK'),
-    next: localize('Next'),
-    skip: localize('Skip'),
-};
+export const getTourStepLocale = (): Locale => ({
```

Comment:
Same comment as above to check if this can be kept as constant and check for language change